### PR TITLE
Debug restart

### PR DESCRIPTION
**Pull request by lawlawlaw**
Fixes an issue in which the mulitisig addresses were not added to the whitelist on restart.

The problem was triggered by a registeraddress script with size > 10 kB. During deserialization (compressor.h) the script was changed to OP_RETURN - this has been disbled if fRequireStandard == false.

Additional checks were added to onboardmanual_cit.py to create and test a registeraddress TX > 11 kB

### CODE REVIEW

File: src/policy/whitelist.cpp [hunk 1]
```diff
@@ -57,7 +57,12 @@ bool CWhiteList::Load(CCoinsView *view)
       CCoins coins;
       if (!(pcursor->GetKey(key) && pcursor->GetValue(coins))) 
         return error("%s: unable to read value", __func__);
-             
+
+      if(RegisterAddress(coins.vout)){
```
Comment (nkostoulas) [inline src/policy/whitelist.cpp]: I don't think this will work. `RegisterAddress` still checks for whitelist asset.

Comment (nkostoulas) [reply src/policy/whitelist.cpp]: I take that back

Comment (nkostoulas) [reply src/policy/whitelist.cpp]: I take that back again, this doesn't work without reindex

Comment (nkostoulas) [reply src/policy/whitelist.cpp]: Tests should fail for registeraddress_v0 type and multisig

Comment (lawlawlaw) [reply src/policy/whitelist.cpp]: Ok, made the necessary changes. Check the edited PR description.

File: src/rpc/misc.cpp [hunk 2]
```diff
@@ -168,7 +168,7 @@ UniValue validateaddress(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() != 1)
         throw runtime_error(
             "validateaddress \"address\"\n"
-            "\nReturn information about the given bitcoin address.\n"
+            "\nReturn information about the giSo ven bitcoin address.\n"
```
Comment (nkostoulas) [inline src/rpc/misc.cpp]: revert please

File: Dockerfile [hunk 3]
```diff
@@ -8,7 +8,7 @@ RUN set -ex \
     && ./autogen.sh \
     && ./configure --without-gui \
     && make clean \
-    && make -j$(nproc) \
+    && make \
```
Comment (nkostoulas) [inline Dockerfile]: revert please

File: qa/rpc-tests/onboardmanual_cit.py [hunk 4]
```diff
@@ -708,9 +707,104 @@ def run_test (self):
...
+            
+        self.nodes[0].generate(6)
+        self.sync_all()
+
+        #Check that the TX size > MAX_SCRIPT_SIZE
+        MAX_SCRIPT_SIZE=10000
+        rawtx=self.nodes[0].getrawtransaction(onboardtx)
+        nchars = len(rawtx)
+        nbytes = nchars/2
+        assert(nbytes > MAX_SCRIPT_SIZE)
```
Comment (nkostoulas) [inline qa/rpc-tests/onboardmanual_cit.py]: It's the scriptpubkey that needs to be >10KB. Maybe that is why you couldn't replicate?

File: Dockerfile (hunk 3 above)
Comment (lawlawlaw) [reply Dockerfile]: Done

File: qa/rpc-tests/onboardmanual_cit.py (hunk 4 above)
Comment (lawlawlaw) [reply qa/rpc-tests/onboardmanual_cit.py]: Latest commits confirm the scriptpubkey size > MAX_SCRIPT_SIZE.

**Pull request merged**